website/content/docs/upgrading/instructions/upgrade-to-1-6-x.mdx

```diff
@@ -202,6 +202,7 @@ update those now to avoid issues when moving to newer versions.
 
 These are the changes you will need to make:
 
+- `acl.enabled` is required (review our [docs](https://www.consul.io/docs/security/acl/acl-system#configuring-acls) for more info)
```

Comment:
```suggestion
- `acl.enabled` is required (review our [docs](/docs/security/acl/acl-system#configuring-acls) for more info)
```
Most of the links on the website use relative URLs rather than absolute URLs.